food-waste-express-api/routes/home.js

```diff
@@ -21,7 +21,6 @@ router.get("/addtobasket", async (req, res) => {
     try {
         let userBasket = await getUserBasket(currentUserId)
         let userBasketOfIds = await userBasket.get("basketOfProductId") //returns first instance
-        console.log("userBasketOfIds", userBasketOfIds)
```

Comment:
Good to clean up these console.logs but please don't include them in your commits. Each commit should be free of console.logs + dead code + minimal comments. This makes it easier to review